src/client/components/leaderboard/leaderboard.js

```diff
@@ -8,8 +8,18 @@ class Leaderboard extends React.Component {
     names: PropTypes.any.isRequired,
     cards: PropTypes.any.isRequired,
     playerID: PropTypes.any.isRequired,
+    passed: PropTypes.any.isRequired,
```

Comment:
I know it's called `passed` in `G`. But maybe it would better readable to rename this property to something like `passedUsers`.

Reply:
We could be more concrete:  `PropTypes.array.isRequired`